Lib/test/test_enum.py

```diff
@@ -4477,6 +4485,19 @@ def test_convert(self):
                           and name not in dir(IntEnum)],
                          [], msg='Names other than CONVERT_TEST_* found.')
 
+    def test_convert_uncomparable(self):
+        uncomp = enum.Enum._convert_(
```

Comment:
Moreover, `_convert_` was not tested on plain `Enum`

Reply:
`_convert_` should never be called on plain `Enum` -- it's purpose it to convert existing constants in a module to enums, and existing constants will already be `str` or `int` (or possibly something else, like `complex`).

Reply:
Change the `UNCOMPABLE_*` values as suggested above, and also add a test using a `complex` type.